app/upgrades/v3/upgrades.go

```diff
@@ -15,14 +17,15 @@ import (
 )
 
 // UpgradeName defines the on-chain upgrade name for the Babylon v3 upgrade
-const UpgradeName = "v3"
-
-const deletedCapabilityStoreKey = "capability"
+const (
+	UpgradeName               = "v3"
+	deletedCapabilityStoreKey = "capability"
+)
 
-func CreateUpgrade() upgrades.Upgrade {
+func CreateUpgrade(fpCount uint32, relativeWaitingTime uint32) upgrades.Upgrade {
```

Comment:
Maybe instead of `relativeWaitingTime`, lets make it possible to pass activation height directly ? (and calculate it offchain)

My main reasoning for doing so, is that it will be easier to handle on some devnets and testing scenarios. 

Wdyt ?